packages/server/ormconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
this file will be generated somehow on pipelines, only for informational purposes here for now